src/main/java/frc/robot/Constants.java

```diff
@@ -1,76 +1,100 @@
+// version 3 as published by the Free Software Foundation or
+// available in the root directory of this project.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
 
-  public static class MotorIDConstants {
+public final class Constants {
+  public static class CtreCanID {
     // Rev Can Bus
+    // 1-8 Motor ID is reserved by the drivebase
     public static final int kElevatorMotor1 = 10;
     public static final int kElevatorMotor2 = 11;
-    public static final int kIntakeMotor = 9;
+    public static final int kPivotMotor = 9; // FIXME: Change to correct ID
+    public static final int kFeedBackSensorID = 0;
+    // public static final int kIntakeMotor = 9;
+    // public static final int kPivotMotor = 0; // FIXME: Change to correct ID
+    // public static final int kConveyorMotor = 0; // FIXME: Change to correct ID
     // CANivore Can Bus
   }
 
+  public static class RevCanID {
+    public static final int kIntakeMotor = 9;
+    public static final int kConveyorMotor = 0; // FIXME: Change to correct ID
+    public static final int kConveyorSensor = 4; // FIXME: Change to correct ID
+  }
+
   // Intake Constants
   public static class IntakeConstants {
-    public static final double kSpeed = 0.5;
+    public static final double kIntakeSpeed = 0.5;
+    public static final double kConveyorSpeed = 0.5;
+    public static final double kPivotSpeed = 0.5;
+    public static final Angle kPivotRetractAngle = Degrees.of(90); // FIXME: Put actual value
+    public static final Angle kPivotExtendAngle =
+        Degrees.of(0); // FIXME: Might have to be negative to reach over bumper
+    public static final Angle kPivotTolerance = Degrees.of(3);
+    public static final double kPivotP = 100;
+    public static final double kPivotI = 0.0;
+    public static final double kPivotD = 0.0;
+    public static final double kPivotG = 0.21;
+    public static final double kPivotV = 0; // 7.20;
+    public static final double kPivotA = 0; // 0.03;
+    public static final GravityTypeValue kPivotGravityType = GravityTypeValue.Arm_Cosine;
+
+    public static final double kGearing = 60;
+    public static final double kSensorToMechanism = 1;
+    public static final Distance kLength = Feet.of(1);
+    public static final Mass kMass = Pounds.of(8);
+    public static final MomentOfInertia kMOI =
+        KilogramSquareMeters.of(
+            SingleJointedArmSim.estimateMOI(kLength.in(Meters), kMass.in(Kilograms)));
+    public static final AngularVelocity kMotionMagicCruiseVelocity =
+        RevolutionsPerSecond.of(999999999);
+    public static final AngularAcceleration kMotionMagicAcceleration =
+        kMotionMagicCruiseVelocity.times(Hertz.of(999999999));
+    public static final double kMotionMagicJerk =
```

Comment:
Umm, why does this number need to be so large, If you just want the max value, you can do Double.MAX_VALUE. That goes for all these 99999999999 numbers too.